rdr_service/genomic/genomic_job_controller.py

```diff
@@ -387,6 +395,25 @@ def ingest_data_files(self, file_path, bucket_name):
         except RuntimeError:
             logging.warning('Inserting data file failure')
 
+    def ingest_informing_loop_records(self, *, loop_type, records):
+
+        if records:
+            logging.info(f'Inserting informing loop for Participant: {records[0].participantId}')
+
+            module_type = [obj for obj in records if obj.fieldName == 'module_type' and obj.valueString]
+            decision_value = [obj for obj in records if obj.fieldName == 'decision_value' and obj.valueString]
```

Comment:
Is this the portion that normalizes the message broker data? Will the items in `records` always be associated? It looks like `get_informing_loop` returns all records for a `message_record_id` and a `loop_type`. Just checking that the `module_type` and `decision_value` records are linked somehow and that there won't be ambiguity if there are more than the expected items in `records`.

Reply:
So `records` is a list, that I confirmed will be no longer than 2 via the tech design payload, that is returned in Message Broker Event Dao and will relate back to record in Event Data model via the `message_record_id`.

Reply:
OK, makes sense.